pkg/utils/kubelessutil.go

```diff
@@ -45,6 +47,15 @@ func EnsureCronJob(client kubernetes.Interface, funcObj *kubelessApi.Function, s
 	} else {
 		timeout, _ = strconv.Atoi(defaultTimeout)
 	}
+
+	schedule := cronjobTriggerObj.Spec.Schedule
+	rawPayload, err := json.Marshal(cronjobTriggerObj.Spec.Payload)
+	payload := string(rawPayload)
```

Comment:
you can just define the payload as string in the struct so you don't need to convert it twice, right? Also that would help for using other formats instead of JSON (like just text)

Reply:
Not really, the payload is going to be defined inside the spec. This is an example:

```
apiVersion: kubeless.io/v1beta1
kind: CronJobTrigger
metadata:
  name: example-index
  namespace: sandbox-sre
  annotations:
    quin.to/squad: sample
    quin.to/tribe: foo
  labels:
    app: example
    function: example-index
    release: release-name
    heritage: Tiller
    chart: serverless-1.0.1
    created-by: drone-helm
    type: serverless
spec:
  function-name: example-index
  schedule: "*/1 * * * *"
  payload:
    morty: smith
    test: true
```

So, the `payload` spec key is actually a `map`. I've tried to parse it as a string, or even other types, but the only way to parse it correctly was with a map :/

Reply:
Not really, the payload is going to be defined inside the spec. This is an example:

```
apiVersion: kubeless.io/v1beta1
kind: CronJobTrigger
metadata:
  name: example-index
  namespace: sandbox-sre
  annotations:
    quin.to/squad: sample
    quin.to/tribe: foo
  labels:
    app: example
    function: example-index
    release: release-name
    heritage: Tiller
    chart: serverless-1.0.1
    created-by: drone-helm
    type: serverless
spec:
  function-name: example-index
  schedule: "*/1 * * * *"
  payload:
    morty: smith
    test: true
```

So, the `payload` spec key is actually a `map`. I've tried to parse it as a string, or even other types, but the only way to parse it correctly was with a map :/